src/ComponentCover.js

```diff
@@ -10,22 +10,28 @@ import { layers } from './theme.js'
  * @example import { ComponentCover } from @dhis2/ui-core
  * @see Live demo: {@link /demo/?path=/story/componentcover--circularloader|Storybook}
  */
-const ComponentCover = ({ children, className, dataTest }) => (
+const ComponentCover = ({ children, className, dataTest, transparent }) => (
     <div className={className} data-test={dataTest}>
         {children}
+        <style jsx>{`
+            div {
+                background: ${transparent
+                    ? 'transparent'
+                    : 'rgba(33, 43, 54, 0.4)'};
+                z-index: ${layers.applicationTop - 1};
```

Comment:
`layers.applicationTop` isn't regarded as a dynamic property since it can be resolved through a constant, so this can live in the second style tag.